app/assets/javascripts/lajoya.js

```diff
@@ -36,7 +36,8 @@
         $('#start-btn').hide();
         $('#meaning').hide();
         $('#warning').show();
-        alert('Favor de usar Chrome, actualizar la página y permitir acceso al micrófono');
+        alert('Favor de usar Chrome, actualizar la página y permitir acceso '\
+          'al micrófono');
```

Comment:
Expected ')' and instead saw 'al micrófono'.<br>Missing semicolon.<br>Expected an identifier and instead saw ')'.<br>Expected an assignment or function call and instead saw an expression.